Kernel/SortHelper.cpp

```diff
@@ -727,7 +727,7 @@ void SortHelper::normaliseSort(VList* qVars, TermList& sort)
   sort = SubstHelper::apply(sort, subst);
 }
 
-void SortHelper::normaliseArgSorts(TermStack& qVars, TermStack& argSorts)
+void SortHelper::normaliseArgSorts(TermStack qVars, TermStack& argSorts)
```

Comment:
Why don't we like passing `qVars` by reference anymore?

Reply:
`qVars` is not modified, but maybe const ref would be better?

Reply:
I'm not sure about size, so let's go for const ref.